src/ripple/core/Config.h

```diff
@@ -159,10 +165,12 @@ class Config : public BasicConfig
     // Validation
     boost::optional<std::size_t> VALIDATION_QUORUM;     // validations to consider ledger authoritative
 
-    std::uint64_t                      FEE_DEFAULT = 10;
-    std::uint64_t                      FEE_ACCOUNT_RESERVE = 200*SYSTEM_CURRENCY_PARTS;
-    std::uint64_t                      FEE_OWNER_RESERVE = 50*SYSTEM_CURRENCY_PARTS;
-    std::uint64_t                      FEE_OFFER = 10;
+    tagged_integer<std::uint64_t, DropsTag>
+                                FEE_DEFAULT{ 10 };
+    tagged_integer<std::uint32_t, DropsTag>
```

Comment:
I'd keep existing types if there's no reason not to. (leave it as `uint64_t`)

secondary to that... is `tagged_integer` `constexpr`?

Reply:
Yes, `tagged_integer` is `constexpr`, but these values in Config.h can be changed by the config file.